### PR TITLE
Create vibe.ms

### PR DESCRIPTION
**Pull request by ZargonKing**

### CODE REVIEW

**Comment by ViceTOP** [comment]
His suit has yellow and red, change color codes?

File: CommandHelper/LocalPackages/commands/buffs/vibe.ms [hunk 1]
```diff
@@ -0,0 +1,34 @@
+@plr = player()
+
+
+
+@hero = associative_array(
+            'name'§eV§ci§eb§ce',
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/vibe.ms]: missing a colon here

File: CommandHelper/LocalPackages/commands/buffs/vibe.ms [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            'val': 'eyJ0aW1lc3RhbXAiOjE0ODE5MDEzNTEyNDYsInByb2ZpbGVJZCI6IjcxM2NhZTdhZWY2ZDRhMDY4OTIyNDc0MTQ1OWMxMmJjIiwicHJvZmlsZU5hbWUiOiJaYXJnb25fS2luZyIsInNpZ25hdHVyZVJlcXVpcmVkIjp0cnVlLCJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvNTA1M2NkNTEyZjIxYWZiMWEyNTg4MTE5MGM0OTU3Y2NjZGMyMTYzMjEyMDM5YjVkYzRiNTgyZTZiMzNjIn19fQ==',
+            'trail': '',
+            'op': 'false',
+            'speech': "I got some good vibes coming your way",
+            'powers': array(
+                "/buff",
+                "/jump 0-3",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/vibe.ms]: need evidence of him jumping high

File: CommandHelper/LocalPackages/commands/buffs/vibe.ms [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+                "/run 0-4",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/vibe.ms]: need evidence of him running fast

File: CommandHelper/LocalPackages/commands/buffs/vibe.ms [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+                "/haste 0-4",
+                "/carryplayer",
+                "/dismountplayer",
+                "/back (In the speedforce)",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/vibe.ms]: he cannot access the speedforce, only disrupt it

File: CommandHelper/LocalPackages/commands/buffs/vibe.ms [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+                "/run 0-1 (In the speedforce)",
+                "/thru (phases through walls)",
+                "/vibrate",
+                "/stopvibrating",
+                "/tp",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/vibe.ms]: need evidence of him teleporting

File: CommandHelper/LocalPackages/commands/buffs/vibe.ms [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+                "/tppos <x y z world>",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/vibe.ms]: need evidence of him teleporting

File: CommandHelper/LocalPackages/commands/buffs/vibe.ms [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+            ),
+            'buffs': array(
+                "/effect ". @plr ." 5 9999 10 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/vibe.ms]: need evidence of his super strength

File: CommandHelper/LocalPackages/commands/buffs/vibe.ms [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+                "/effect ". @plr ." 10 9999 1 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/vibe.ms]: need evidence of his regen ability

File: CommandHelper/LocalPackages/commands/buffs/vibe.ms [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+                "/effect ". @plr ." 21 9999 9 true",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/vibe.ms]: need evidence of higher than normal health

File: CommandHelper/LocalPackages/commands/buffs/vibe.ms [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+                "/shot give ". @plr ." kryptonianlaservision",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/vibe.ms]: he doesnt get kryptonian laser vision

File: CommandHelper/LocalPackages/commands/buffs/vibe.ms [hunk 11]
```diff
@@ -0,0 +1,29 @@
+@plr = player()
+
+
+
+@hero = associative_array(
+            'name: '§eV§ci§eb§ce',
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/vibe.ms]: this line is missing an apostrophe

'

File: CommandHelper/LocalPackages/commands/buffs/vibe.ms [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+            'powers': array(
+                "/buff",
+                "/jump 0-2",
+                "/run 0-2",
+                "/haste 0-2",
+                "/carryplayer",
+                "/dismountplayer",
+                "/vibrate",
+                "/stopvibrating",
+                "/tp",
```
Comment (malonnnn) [inline CommandHelper/LocalPackages/commands/buffs/vibe.ms]: remove this he only gets tppos

**Pull request merged**